ext/TrixiConvexECOSExt.jl

```diff
@@ -166,6 +216,100 @@ function Trixi.bisect_stability_polynomial(consistency_order, num_eig_vals,
         gamma_opt = [gamma_opt]
     end
 
+    undo_normalization!(gamma_opt, num_stage_evals,
+                        consistency_order, consistency_order)
+
+    return gamma_opt, dt
+end
+
+# Specialized routine for PERK4.
+# For details, see Section 4 in 
+# - D. Doehring, L. Christmann, M. Schlottke-Lakemper, G. J. Gassner and M. Torrilhon (2024).
+# Fourth-Order Paired-Explicit Runge-Kutta Methods
+# [DOI:10.48550/arXiv.2408.05470](https://doi.org/10.48550/arXiv.2408.05470)
+function Trixi.bisect_stability_polynomial_PERK4(num_eig_vals,
+                                                 num_stage_evals,
+                                                 dtmax, dteps, eig_vals, cS3;
+                                                 verbose = false)
+    dtmin = 0.0
+    dt = -1.0
+    abs_p = -1.0
+
+    # Construct stability polynomial for each eigenvalue
+    pnoms = ones(Complex{Float64}, num_eig_vals, 1)
+
+    # Init datastructure for monomial coefficients
+    gamma = Variable(num_stage_evals - 5)
+
+    normalized_powered_eigvals = zeros(Complex{Float64}, num_eig_vals, num_stage_evals)
+
+    for j in 1:num_stage_evals
+        fac_j = factorial(j)
+        for i in 1:num_eig_vals
+            normalized_powered_eigvals[i, j] = eig_vals[i]^j / fac_j
+        end
+    end
```

Comment:
These lines of code are the same as lines 140-145. If we don't want to merge this function with `bisect_stability_polynomial`, could we maybe try to minimize the lines of code that were written twice?

we can create maybe a short helper function
```
@inline function normalize_powered_eigenvalues!(normalized_powered_eigvals, num_eigvals, num_stage_eval)
  for j in 1:num_stage_evals
          fac_j = factorial(j)
          for i in 1:num_eig_vals
              normalized_powered_eigvals[i, j] = eig_vals[i]^j / fac_j
          end
      end
end
```